refrepo_git.py

```diff
@@ -168,7 +169,7 @@ def extract_remote_conf(remotes):
         parts = remote.split()
         if parts[2] == "(fetch)":
             url = parts[1]
-            if os.getenv(_DROP_CREDENTIALS_ENV_VAR):
+            if os.getenv(_DROP_CREDENTIALS_ENV_VAR).lower() in _TRUTHY_VALUES:
```

Comment:
I tested this (ish) and it fails if the env var is undefined, since `getenv` returns `None` rather than an empty string when the environment variable is undefined.

```
>>> print(os.getenv("NONEXISTENTENVVAR").lower())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'lower'
```

Reply:
Ah, my bad. I'll make it default to "False".